content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -322,6 +322,179 @@ the "black" and "red" ratings in the "Book Reviews" section.
 
 > There may be some delays due to caching and other propagation overhead.
 
+## RBAC policy permissive mode
+
+RBAC permissive mode is for testing a new RBAC policy works as expected before
+rolling out to production. This sections shows how RBAC permissive mode is used.
+
+### Permissive mode for global RBAC configuration
+
+Before you start, please make sure that you have finished [preparation task](#before-you-begin).
+
+1.  Before enabling RBAC in production, apply the global RBAC configuration in
+    permissive mode.
+
+    Run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f samples/bookinfo/platform/kube/rbac/rbac-config-on-permissive.yaml
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`), you should
+    see everything works fine, same as in [preparation task](#before-you-begin).
+
+1.  Apply YAML file for the permissive mode metric collection.
+
+    Run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml
+    logentry.config.istio.io/newlog created
+    stdio.config.istio.io/newhandler created
+    rule.config.istio.io/newlogstdio created
+    {{< /text >}}
+
+1.  Send traffic to the sample application.
+
+    For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+1.  Verify that the logs stream has been created and check `permissiveResponseCode`.
+
+    In a Kubernetes environment, search through the logs for the istio-telemetry
+    pod as follows:
+
+    {{< text bash json >}}
+    $ kubectl -n istio-system logs $(kubectl -n istio-system get pods -l istio-mixer-type=telemetry -o jsonpath='{.items[0].metadata.name}') -c mixer | grep \"instance\":\"newlog.logentry.istio-system\"
+    {"level":"warn","time":"2018-08-29T02:17:17.619428Z","instance":"newlog.logentry.istio-system","destination":"reviews","latency":"6.055691ms","responseCode":200,"responseSize":295,"permissiveResponseCode":"403","permissiveResponsePolicyID":"","source":"productpage","user":"cluster.local/ns/default/sa/bookinfo-productpage"}
+    {"level":"warn","time":"2018-08-29T02:17:17.605665Z","instance":"newlog.logentry.istio-system","destination":"productpage","latency":"26.601231ms","responseCode":200,"responseSize":4415,"permissiveResponseCode":"403","permissiveResponsePolicyID":"","source":"istio-ingressgateway","user":"cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"}
+    {{< /text >}}
+
+    In telemetry logs above,  the `responseCode` is 200 which is what user see now.
```

Comment:
Can we have permissive for role binding together with this (permissive) rbac enable? It is better example to show how to enable RBAC without breaking service.

Reply:
updated the example as 
1. apply global rbac config in permissive mode, got 403 in telemetry logs 
2. on top of 1, apply a policy in permissive mode, got 200 in telemetry logs